code/src/components/Survey.js

```diff
@@ -0,0 +1,206 @@
+      )}
+
+      {counter === 3 && (
+        <div className="survey-info">
+          <UserFeelings
+            userFeelings={userFeelings}
+            setFeelings={setUserFeelings}
+            userName={userName}
+            userTeam={userTeam}
+          />
+        </div>
+      )}
+
+      {counter === 4 && (
+        <div className="survey-info">
+          <UserAloneOrBuddy
+            userAloneOrBuddy={userAloneOrBuddy}
+            setUserAloneOrBuddy={setUserAloneOrBuddy}
+            userName={userName}
+            onEnter={onEnter}
+          />
+        </div>
+      )}
+
+      {(counter === 5 && userAloneOrBuddy === 'alone') && (
+        <div className="survey-info">
+          <UserHours
+            userHours={userHours}
+            setUserHours={setUserHours}
+            onEnter={onEnter}
+          />
+        </div>
+      )}
+
+      {(counter === 5 && userAloneOrBuddy === 'pair') && (
+        <div className="survey-info">
+          <UserInput
+            userInput={userBuddy}
+            setUserInput={setUserBuddy}
+            headerText="The power of 2 minds!"
+            subHeaderText="Who was your code pair?"
+            onEnter={onEnter}
+            placeHolder="Type the name of your code pair"
+          />
+        </div>
+      )}
+
+      {(counter === 5 && userAloneOrBuddy === '') && (
+        setCounter(counter + 1)
+      )}
+
+      {counter === 6 && (
+        <div className="answers-wrapper">
+          <UserConfirm />
+        </div>
+      )}
+
+      {counter === 7 && (
+        <>
+          <div>
+            <UserAnswers
+              userName={userName}
+              userTeam={userTeam}
+              userFeelings={userFeelings}
+              userBuddy={userBuddy}
+              userAloneOrBuddy={userAloneOrBuddy}
+              userHours={userHours}
+            />
+          </div>
+          <div className="btn-container">
+            <button
+              className="btn restart"
+              type="button"
+              onClick={handleRestartButtonClick}>
+              Restart
+            </button>
+          </div>
+        </>
+      )}
+
+      {(counter >= 1 && counter <= 5) && (
+        <div className="btn-container">
+          <button
+            className="btn back"
+            type="button"
+            onClick={() => handleCounterButtonClick(false)}>
+            Back
+          </button>
+          <button
+            className="btn next"
+            type="button"
+            onClick={() => handleCounterButtonClick(true)}>
+            Next
+          </button>
+        </div>
+      )}
+
+      {counter === 6 && (
+        <div className="btn-container">
+          <button
+            className="btn submit"
+            type="submit"
+            onClick={handleSubmitButtonClick}>
+            Submit
+          </button>
+          <button
+            className="btn restart"
+            type="button"
+            onClick={handleRestartButtonClick}>
```

Comment:
I tip here could perhaps be to make the button a reusable component what you can pass different props to depending on what kind of button you need (go back, continue etc). But this works very well too! :)